diff --git a/vendor/magento/module-inventory-source-deduction-api/Model/SourceDeductionService.php b/vendor/magento/module-inventory-source-deduction-api/Model/SourceDeductionService.php
index f837fe2ca..a3dd6bc14 100644
--- a/vendor/magento/module-inventory-source-deduction-api/Model/SourceDeductionService.php
+++ b/vendor/magento/module-inventory-source-deduction-api/Model/SourceDeductionService.php
@@ -85,7 +85,7 @@ class SourceDeductionService implements SourceDeductionServiceInterface
             }
 
             $sourceItem = $this->getSourceItemBySourceCodeAndSku->execute($sourceCode, $itemSku);
-            if (($sourceItem->getQuantity() - $qty) >= 0) {
+            if (($sourceItem->getQuantity() - $qty) >= 0 || $stockItemConfiguration->getBackorders()) {
                 $sourceItem->setQuantity($sourceItem->getQuantity() - $qty);
                 $stockStatus = $this->getSourceStockStatus(
                     $stockItemConfiguration,
