diff --git a/vendor/aheadworks/module-automatic-related-products/Block/Related/Block.php b/vendor/aheadworks/module-automatic-related-products/Block/Related/Block.php
index 39ea7a980..8f2001de2 100644
--- a/vendor/aheadworks/module-automatic-related-products/Block/Related/Block.php
+++ b/vendor/aheadworks/module-automatic-related-products/Block/Related/Block.php
@@ -206,7 +206,7 @@ class Block extends ElementTemplate
      */
     public function jsonEncode($params)
     {
-        return \Zend_Json::encode($params);
+        return \Laminas\Json\Json::encode($params);
     }
     /**
      * Retrieve corresponding rule id for specific block
