diff --git a//Model/Parser.php b//Model/Parser.php
index e90bea423..c9d7c41b8 100644
--- a//Model/Parser.php
+++ b//Model/Parser.php
@@ -76,7 +76,7 @@ class Parser implements ParserInterface
      * @param ParserContextInterface $context
      * @param string $configuration
      */
-    public function parse(ParserContextInterface $context, string $configuration): void
+    public function parse(ParserContextInterface $context, ?string $configuration): void
     {
         $context->getDebug() && $this->debugLogger->collapseOpen($context->getDebugPrefix());
 
