diff --git a/view/base/web/js/widget/video-background.js b/view/base/web/js/widget/video-background.js
index 61fdf615b..88669d151 100644
--- a/view/base/web/js/widget/video-background.js
+++ b/view/base/web/js/widget/video-background.js
@@ -3,10 +3,7 @@
  * See COPYING.txt for license details.
  */
 define([
-    'jquery',
-    'jarallax',
-    'jarallaxVideo',
-    'vimeoWrapper'
+    'jquery'
 ], function ($) {
     'use strict';

@@ -21,20 +18,25 @@ define([
         $element.addClass('jarallax');
         $element.attr('data-jarallax', '');

-        window.jarallax($element[0], {
-            imgSrc: $element.data('videoFallbackSrc'),
-            speed: !isNaN(parallaxSpeed) ? parallaxSpeed : 0.5,
-            videoLoop: $element.data('videoLoop'),
-            videoPlayOnlyVisible: $element.data('videoPlayOnlyVisible'),
-            videoLazyLoading: $element.data('videoLazyLoad'),
-            disableVideo: false,
-            elementInViewport: $element.data('elementInViewport') &&
-                $element[0].querySelector($element.data('elementInViewport'))
-        });
-        $element[0].jarallax.video && $element[0].jarallax.video.on('started', function () {
-            if ($element[0].jarallax.$video) {
-                $element[0].jarallax.$video.style.visibility = 'visible';
-            }
+        require(['jarallax', 'jarallaxVideo', 'vimeoWrapper'], function () {
+            $element.addClass('jarallax');
+            $element.attr('data-jarallax', '');
+
+            window.jarallax($element[0], {
+                imgSrc: $element.data('videoFallbackSrc'),
+                speed: !isNaN(parallaxSpeed) ? parallaxSpeed : 0.5,
+                videoLoop: $element.data('videoLoop'),
+                videoPlayOnlyVisible: $element.data('videoPlayOnlyVisible'),
+                videoLazyLoading: $element.data('videoLazyLoad'),
+                disableVideo: false,
+                elementInViewport: $element.data('elementInViewport') &&
+                    $element[0].querySelector($element.data('elementInViewport'))
+            });
+            $element[0].jarallax.video && $element[0].jarallax.video.on('started', function () {
+                if ($element[0].jarallax.$video) {
+                    $element[0].jarallax.$video.style.visibility = 'visible';
+                }
+            });
         });
     };
 });
\ No newline at end of file
