diff --git a/vendor/magento/module-theme/view/base/requirejs-config.js b/vendor/magento/module-theme/view/base/requirejs-config.js
index 28495b628cd3..7e4b2da8b757 100644
--- a/vendor/magento/module-theme/view/base/requirejs-config.js
+++ b/vendor/magento/module-theme/view/base/requirejs-config.js
@@ -86,6 +86,11 @@ var config = {
             'headers': {
                 'X-Requested-With': 'XMLHttpRequest'
             }
+        },
+        mixins: {
+            'jquery': {
+                'jquery/jquery-scroll-passive-patch': true
+            }
         }
     }
 };