diff --git a/vendor/mirasvit/module-email-designer/src/EmailDesigner/Service/TemplateEngine/Liquid/VariableProvider.php b/vendor/mirasvit/module-email-designer/src/EmailDesigner/Service/TemplateEngine/Liquid/VariableProvider.php
index 846acc6ab3..6de6339e5f 100644
--- a/vendor/mirasvit/module-email-designer/src/EmailDesigner/Service/TemplateEngine/Liquid/VariableProvider.php
+++ b/vendor/mirasvit/module-email-designer/src/EmailDesigner/Service/TemplateEngine/Liquid/VariableProvider.php
@@ -229,7 +229,7 @@ class VariableProvider implements VariableProviderInterface
                 'value' => $value,
                 'label' => $docblock->hasTag(VariableInterface::DOCBLOCK_TAG_DESCRIPTION)
                     ? $docblock->getTag(VariableInterface::DOCBLOCK_TAG_DESCRIPTION)->getDescription()
-                    : explode("\n", $docblock->getShortDescription())[0]
+                    : explode("\n", (string) $docblock->getShortDescription())[0]
             ];
         }