diff --git a/vendor/magento/module-inventory-sales/Model/IsProductSalableCondition/BackOrderNotifyCustomerCondition.php b/vendor/magento/module-inventory-sales/Model/IsProductSalableCondition/BackOrderNotifyCustomerCondition.php
index 813679b4..3f619130 100644
--- a/vendor/magento/module-inventory-sales/Model/IsProductSalableCondition/BackOrderNotifyCustomerCondition.php
+++ b/vendor/magento/module-inventory-sales/Model/IsProductSalableCondition/BackOrderNotifyCustomerCondition.php
@@ -86,8 +86,7 @@ class BackOrderNotifyCustomerCondition implements IsProductSalableForRequestedQt
                     $this->productSalabilityErrorFactory->create([
                         'code' => 'back_order-not-enough',
                         'message' => __(
-                            'We don\'t have as many quantity as you requested, '
-                            . 'but we\'ll back order the remaining %1.',
+                            "We do not have as many quantity as you requested, but we'll back order the remaining %1.",
                             $backorderQty * 1
                         )])
                 ];
