diff --git a/view/base/web/js/widget/video-background.js b/view/base/web/js/widget/video-background.js
index d918ab8f6..e42c001f1 100644
--- a/view/base/web/js/widget/video-background.js
+++ b/view/base/web/js/widget/video-background.js
@@ -18,23 +18,26 @@ define([
             return;
         }
 
-        $element.addClass('jarallax');
-        $element.attr('data-jarallax', '');
+        require(['jarallax', 'jarallaxVideo', 'vimeoWrapper'], function () {
+            $element.addClass('jarallax');
+            $element.attr('data-jarallax', '');
 
-        window.jarallax($element[0], {
-            imgSrc: $element.data('videoFallbackSrc'),
-            speed: !isNaN(parallaxSpeed) ? parallaxSpeed : 0.5,
-            videoLoop: $element.data('videoLoop'),
-            videoPlayOnlyVisible: $element.data('videoPlayOnlyVisible'),
-            videoLazyLoading: $element.data('videoLazyLoad'),
-            disableVideo: false,
-            elementInViewport: $element.data('elementInViewport') &&
-                $element[0].querySelector($element.data('elementInViewport'))
-        });
-        $element[0].jarallax.video && $element[0].jarallax.video.on('started', function () {
-            if ($element[0].jarallax.$video) {
-                $element[0].jarallax.$video.style.visibility = 'visible';
-            }
+            window.jarallax($element[0], {
+                imgSrc: $element.data('videoFallbackSrc'),
+                speed: !isNaN(parallaxSpeed) ? parallaxSpeed : 0.5,
+                videoLoop: $element.data('videoLoop'),
+                videoPlayOnlyVisible: $element.data('videoPlayOnlyVisible'),
+                videoLazyLoading: $element.data('videoLazyLoad'),
+                disableVideo: false,
+                elementInViewport: $element.data('elementInViewport') &&
+                    $element[0].querySelector($element.data('elementInViewport'))
+            });
+            
+            $element[0].jarallax.video && $element[0].jarallax.video.on('started', function () {
+                if ($element[0].jarallax.$video) {
+                    $element[0].jarallax.$video.style.visibility = 'visible';
+                }
+            });
         });
     };
 });
